sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveExplainSuite.scala

```diff
@@ -27,6 +27,21 @@ import org.apache.spark.sql.test.SQLTestUtils
  */
 class HiveExplainSuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
 
+  test("show stats in explain command") {
+    withSQLConf("spark.sql.statistics.showInExplain" -> "false") {
+      checkKeywordsNotExist(sql(" explain  select * from src "), "sizeInBytes", "rowCount")
```

Comment:
A general style suggestion. Normally, the SQL keywords are using upper case in the test cases.

`explain  select * from src` -> `EXPLAIN SELECT * FROM src`